src/api-endpoints.ts

```diff
@@ -4681,6 +4683,13 @@ type LastEditedTimeDatabasePropertyConfigResponse = {
   name: string
 }
 
+type ButtonDatabasePropertyConfigResponse = {
```

Comment:
@hazelguo this looks handwritten - if you don't add this, what happens to the types in this file? we shouldn't be exposing this property in the SDK yet (and probably not ever tbh)

Reply:
I didn't handwrite anything. Everything is from `notion build-public-api-js-sdk --path=../notion-sdk-js/src/`